themes/default/index.html

```diff
@@ -248,8 +248,6 @@
                   <td class="content-cell">
                     <% if (locals.title) { %>
```

Comment:
Removed. Perhaps it would be nice to allow no output of this title if no `title` or `name` is present. One for a different PR though 👍